src/app/profiles/store/profile.reducer.ts

```diff
@@ -0,0 +1,35 @@
+
+export interface ProfileFormState {
+  tv: string;
+  streaming: object;
+  city: string;
+  address: string;
+  postal: string;
+  home: string;
+  antenna: string;
+  aptfloor: string;
```

Comment:
Does this mean apartment floor? I might just naming it so, instead of abbreviating.

Reply:
Yes, its used to estimate the antenna height from ground level.  I will consider de-abbreviating.